components/brave_news/browser/resources/Feed.tsx

```diff
@@ -122,7 +122,9 @@ export default function Component({ feed, onSessionStart }: Props) {
   const lastViewedCardCount = React.useRef(0);
   const viewDepthIntersectionObserver = React.useRef(new IntersectionObserver(entries => {
     const inViewCounts = entries
-      .filter(e => e.intersectionRatio === 1)
+      // Check greater than 0 intersection, as when the top of the next card
+      // shows, we've seen the current one.
+      .filter(e => e.intersectionRatio > 0)
```

Comment:
any reason why this can't be `e.intersectionRation === 1` ?

Reply:
it seems that we should be able to check if the current card is in full-view, rather than the top of the next card

Reply:
Yeah, might as well - and it means we wouldn't need to do the little rain dance to put the count on the next item.